Tchap/Managers/ThirdPartyIDResolver/ThirdPartyIDResolver.swift

```diff
@@ -40,9 +40,9 @@ final class ThirdPartyIDResolver: NSObject, ThirdPartyIDResolverType {
         guard let identityServerURL = URL(string: identityServer) else {
             return nil
         }
-        
+        let accessToken = session.credentials.accessToken
         let identityService = MXIdentityService(identityServer: identityServerURL,
-                                                accessToken: nil,
+                                                accessToken: accessToken,
```

Comment:
You could not provide this access token here

Reply:
IdentityService instance has been removed here.